awsconfigs/common/aws-alb-ingress-controller/base/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+# Application Load Balancer (ALB) Ingress Controller Deployment Manifest.
+# This manifest details sensible defaults for deploying an ALB Ingress Controller.
+# GitHub: https://github.com/kubernetes-sigs/aws-alb-ingress-controller
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: alb-ingress-controller
+  # Namespace the ALB Ingress Controller should run in. Does not impact which
+  # namespaces it's able to resolve ingress resource for. For limiting ingress
+  # namespace scope, see --watch-namespace.
+#  namespace: kubeflow
```

Comment:
Why is the namespace commented out?

Reply:
conflicts with patch. removed it altogether